### ISSUE TITLE
require Crypt/RSA.php line missing?

### ISSUE THREAD

**Issue by JanOonk** [jumbojett/OpenID-Connect-PHP#142]
I am using the latest version v0.7.
I didn't use composer, just copied and included the library in my project. But I had to add this line before the class_exists lines to get the library up and running:
```
require "Crypt/RSA.php";

if (!class_exists('\phpseclib\Crypt\RSA') && !class_exists('Crypt_RSA')) {
    user_error('Unable to find phpseclib Crypt/RSA.php.  Ensure phpseclib is installed and in include_path before you include this file');
}
```

I copied the phpseclib subdirectory from the src installation package into my lib folder.
 In my application I then use the openidconnect library as follows:
```
  set_include_path(get_include_path().PATH_SEPARATOR.'./lib/phpseclib'.PATH_SEPARATOR.'./lib/OpenID-Connect-PHP');
  
  // Include the openID client
  require "OpenIDConnectClient.php";
  use Jumbojett\OpenIDConnectClient;

  // Create a new client
  $oidc = new OpenIDConnectClient(OPENID_SERVER,FHICT_CLIENT_ID,FHICT_CLIENT_SECRET);
```

Is this fix an omission in your source or am I doing something wrong?

**Comment by jumbojett** [comment]
> I didn't use composer, just copied and included the library in my project. 

@JanOonk composer will pull in a the dependancies! You should definitely be using it.

**Comment by JanOonk** [comment]
That I know but that still doesn't answer my question.

**Comment by jumbojett** [comment]
> Is this fix an omission in your source or am I doing something wrong?

Apologies. I'm not sure I fully understand. Composer includes an autoloader that configures path dependencies. Can you rephrase the question?

**Comment by JanOonk** [comment]
For all people that can not use composer and just like me, in this instance, won't use composer and just follow the procedure I described. The fact that I had to insert `require "Crypt/RSA.php";` is that a bug/omission in the source?

**Comment by jumbojett** [comment]
In the case you describe, it's an omission. Not a bug.

**Comment by JanOonk** [comment]
Ok not a bug, it's more like a dependency missing. Consider it documented by this issue :)
